### ISSUE TITLE
I have recently implemented the RazorPay in my flutter app. I have use this plugin => https://pub.dev/packages/razorpay_flutter & implemented as given in documentation. But  once payment is success I getting response like this => {razorpay_signature: null, razorpay_order_id: null, razorpay_payment_id: pay_OH1mKQOtest}.

### ISSUE THREAD

**Issue by Hasnainhai** [razorpay/razorpay-flutter#385]
I have recently implemented the RazorPay in my flutter app. I have use this plugin => https://pub.dev/packages/razorpay_flutter & implemented as given in documentation. But  once payment is success I getting response like this => {razorpay_signature: null, razorpay_order_id: null, razorpay_payment_id: pay_OH1mKQOtest}.

So here why I'm getting this razorpay_order_id = null?

PS.I'm using razorpay in TEST mode.
 Same error please anyone can help us

**Comment by vivekshindhe** [comment]
@Hasnainhai Are you passing the order_id in the options payload which are submitted to the razorpay_flutter package?

**Comment by Hasnainhai** [comment]
@vivekshindhe 
 yes i have provided the order id which is give in the api i get the orderid from the api and pass it to razor_pay options but i didn't get the order Id and signature in return.

**Comment by Jackjohn123456** [comment]
@Hasnainhai  Same Issue with me, I am also using test razor pay api key.
I have attached debug image for reference. Is it default behaviour from razorPay ? 

<img width="1058" height="578" alt="Image" src="https://github.com/user-attachments/assets/47116fa1-b01a-4185-89e8-ee4b3b4c5eab" />

**Comment by Jackjohn123456** [comment]
> [@Hasnainhai](https://github.com/Hasnainhai) Same Issue with me, I am also using test razor pay api key. I have attached debug image for reference. Is it default behaviour from razorPay ?
> 
> <img alt="Image" width="1058" height="578" src="https://private-user-images.githubusercontent.com/101467989/547873097-47116fa1-b01a-4185-89e8-ee4b3b4c5eab.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzA4ODE5NzgsIm5iZiI6MTc3MDg4MTY3OCwicGF0aCI6Ii8xMDE0Njc5ODkvNTQ3ODczMDk3LTQ3MTE2ZmExLWIwMWEtNDE4NS04OWU4LWVlNGIzYjRjNWVhYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDIxMlQwNzM0MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jOThkZjJjMTkxZWJkNjI5MWU3NTE2ZGJhZmU5NGMxNzUzYmZlNDQxNTA2YzMzMmQ1MmVmMjJmNWExMTcxNjM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.iVocQHDY9Ev7jqX5wB6f-CKpoiJ8y0SrC9GeRVU3CTU">

I Found My Error, My FomJson returned "orderId " instead of "order_id" field. Hence payment_id not asscoiates to order_id resulting in no signature. May this finds helpful : )
